packages/embed/src/utils/remove-undefined-keys.ts

```diff
@@ -0,0 +1,5 @@
+export const removeUndefinedKeys = (obj: Record<string, any>): Record<string, any> => {
+  return Object.entries(obj)
+    .filter(([_key, value]) => value !== undefined && value !== null)
```

Comment:
```suggestion
    .filter(([, value]) => value != null)
```

Reply:
I know `!= null`  catches only `null` and `undefined` but I really hate how non-strict comparison in javascript can lead to unexpected results. Not really in this case, but you need to have deeper understanding of the language. Shorthands like this also lead to "why not just `!value`?" which also catches empty string and zero - and thats obviously incorrect.